process/transaction/shardProcess.go

```diff
@@ -970,6 +982,10 @@ func (txProc *txProcessor) processUserTx(
 	switch txType {
 	case process.MoveBalance:
 		err = txProc.processMoveBalance(userTx, acntSnd, acntDst, dstShardTxType, originalTxHash, true)
+		isUserTxOfRelayedV3 := len(originalTx.InnerTransactions) > 0
```

Comment:
this if can be deleted and add a complete event log for each such processing.

Reply:
But actually the completeEventLog should be done only if destination shard is the current shard.

otherwise you may have execution on the destination shard and create another log event.

so add log event is this is intra shard only. otherwise, completeLogEvent is added by scProcessor. plus add an integration test and verify how many completeLogEvents are generated.